core/server/worker/src/test/java/alluxio/worker/block/BlockHeartbeatReporterTest.java

```diff
@@ -95,27 +102,27 @@ public void generateReportMove() {
   }
 
   /**
-   * Tests the {@link BlockHeartbeatReporter#generateReport()} method that generating a report
-   * clears the state of the reporter.
+   * Tests the {@link BlockHeartbeatReporter#generateReportAndClear()}
+   * method that generating a report clears the state of the reporter.
    */
   @Test
   public void generateReportStateClear() {
     Long block1 = 1L;
     moveBlock(block1, MEM_LOC);
 
     // First report should have updates
-    BlockHeartbeatReport report = mReporter.generateReport();
+    BlockHeartbeatReport report = mReporter.generateReportAndClear();
     assertFalse(report.getAddedBlocks().isEmpty());
 
     // Second report should not have updates
```

Comment:
No change required here.

Actually we prefer the master-side to be idempotent, and in the near future we should allow the worker to send dup updates. We might even prefer the worker to do so, because losing updates may result in mysterious bugs. So i'm okay if we later break this test and remove it.

Reply:
we changed nothing... we just renamed the method generateReport() to generateReportAndClear()